### DISCUSSION TITLE
View application from remote PC

### DISCUSSION THREAD

**Discussion by CMDRZOD** [ElementsProject/umbrel-core-lightning#62] [Q&A]
Hi. I can't seem to figure out how to bind this application to my LAN address. I am currently running it on a headless PC with my node. I am attempting to access the page from another PC on my network. Whatever I try the application only seems to bind to the localhost and I am unable to access the application from my network PC.

How can I set the application to bind to my LAN IP? Thanks.

From the command line:
{"level":"warn","message":"\tServer running at http://localhost:2103","timestamp":"2023-12-17 18:35:35:3535"}

**Comment by ShahanaFarooqui** [comment]
Environment variable `APP_CORE_LIGHTNING_IP` can be used to configure it on your headless PC's IP, like:
`export APP_CORE_LIGHTNING_IP="192.168.1.85"`
